### PR TITLE
fix: #tno-1576 - hide aliases on my ministers page

### PR DESCRIPTION
**Pull request by kylgarmor**
- also make minister name bold

### CODE REVIEW

File: app/subscriber/src/features/settings/MyMinisterSettings.tsx [hunk 1]
```diff
@@ -15,7 +15,7 @@ export const MyMinisterSettings: React.FC = () => {
 
   let activeMinisters = ministers
     .filter((m) => m.isEnabled)
-    .sort((a, b) => a.name.localeCompare(b.name));
+    .sort((a, b) => a.sortOrder - b.sortOrder);
```
Comment (Fosol) [inline app/subscriber/src/features/settings/MyMinisterSettings.tsx:18]: Wouldn't this only sort by the `sortOrder`?  Doesn't the other function also sort by the `name`?

File: app/subscriber/src/features/settings/MyMinisterSettings.tsx [hunk 2]
```diff
@@ -84,7 +88,7 @@ export const MyMinisterSettings: React.FC = () => {
                   }
                 }}
               />
-              <div className="aliases">Search Aliases: {o.aliases}</div>
+              <div className="position">{o.position}</div>
```
Comment (Fosol) [inline app/subscriber/src/features/settings/MyMinisterSettings.tsx:91]: Was this requested?  I don't think we have the `position` data presently, which would result in this being blank.

File: app/subscriber/src/features/settings/MyMinisterSettings.tsx [hunk 3]
```diff
@@ -15,7 +15,7 @@ export const MyMinisterSettings: React.FC = () => {
 
   let activeMinisters = ministers
     .filter((m) => m.isEnabled)
-    .sort((a, b) => a.name.localeCompare(b.name));
+    .sort((a, b) => a.sortOrder - b.sortOrder || a.name.localeCompare(b.name));
```
Comment (Fosol) [inline app/subscriber/src/features/settings/MyMinisterSettings.tsx:18]: Won't this ignore `sortOrder` if `=-1 || 0` and then use the `name` instead?  The `sortOrder` should be the priority sort`.

File: app/subscriber/src/features/settings/MyMinisterSettings.tsx (hunk 2 above)
Comment (kylgarmor) [reply app/subscriber/src/features/settings/MyMinisterSettings.tsx:91]: Yah we have `position` data now.  Carolynn has been entering it.

When you are back next week we can talk about the `description` field on the `ministers` table.  It doesn't have a purpose currently, so could arguably be hidden in the Editor UI.
Also, the Organization table/relationship you added before you left isn't quite that straight forward apparently.  Carolynn showed me that there are cases where a minister can have multiple parent orgs :(

Comment (Fosol) [reply app/subscriber/src/features/settings/MyMinisterSettings.tsx:91]: They don't have any of this functionality in the existing product, so we should limit how much we add for MVP.

File: app/subscriber/src/features/settings/MyMinisterSettings.tsx (hunk 3 above)
Comment (kylgarmor) [reply app/subscriber/src/features/settings/MyMinisterSettings.tsx:18]: I honestly wasn't too sure how that worked either, but it does..
image

image

**Review by Fosol — APPROVED** [review]

**Pull request merged**